product_pricelist_auto_mailing/models/product_pricelist_item.py

```diff
@@ -0,0 +1,13 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = 'product.pricelist.item'
+
+    def write(self, vals_list):
```

Comment:
Don't use `vals_list`, as suggests a list, but this parameter is a dictionary (`vals_list` is reserved for a multi `create` method):
```suggestion
    def write(self, vals):
```